clang/test/CodeGenSYCL/ptx-reqd-work-group-size.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang_cc1 -fsycl-is-device %s -emit-llvm -triple nvptx64-nvidia-cuda-sycldevice -o - | FileCheck %s
+
+template <typename name, typename Func>
```

Comment:
Please `#include "sycl.hpp"`. You can use the headers included in test folder by adding `-internal-isystem %S/Inputs` to RUN line.

Also, please add a comment describing the test